tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -138,7 +138,7 @@ def tap_switch_to_gallery(self):
         switch_to_gallery_button.tap()
         from gaiatest.apps.gallery.app import Gallery
         gallery_app = Gallery(self.marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == gallery_app.name)
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == gallery_app.manifest_url)
```

Comment:
instead of this https://github.com/mozilla-b2g/gaia/pull/31259#discussion-diff-37988317R141